addon/components/pix-tooltip.js

```diff
@@ -18,4 +21,23 @@ export default class PixTooltip extends Component {
   get display() {
     return typeof this.args.hide === 'undefined' || !this.args.hide;
   }
+
+  @action
+  hiddenTooltip(event) {
+    event.target.classList.remove('pix-tooltip--visible');
+  }
+
+  @action
+  mouseOutTooltip(event) {
+    const isFocused = document.activeElement === event.target;
+    if (isFocused) {
+      return;
+    }
+    event.target.classList.remove('pix-tooltip--visible');
+  }
+
+  @action
+  displayTooltip(event) {
```

Comment:
```suggestion
  showTooltip(event) {
```

Je trouve ça juste un peu plus explicite

Reply:
Je suis d'accord !